docs/src/content/docs/tr/guides/components.mdx

```diff
@@ -65,18 +65,66 @@ Her `<TabItem>` bileşenini kullanıcılara göstermek için `label` sahibi olma
 import { Tabs, TabItem } from '@astrojs/starlight/components';
 
 <Tabs>
-	<TabItem label="Yıldızlar">Sirius, Vega, Betelgeuse</TabItem>
-	<TabItem label="Uydular">Io, Europa, Ganymede</TabItem>
+	<TabItem label="Yıldızlar" icon="star">Sirius, Vega, Betelgeuse</TabItem>
```

Comment:
I cannot leave a comment for the sentences above the example but it looks like this paragraph contains a few issues:

- The "You can display a tabbed interface using the `<Tabs>` and `<TabItem>` components." is in English.
- It's missing the `icon` attribute description and link

Would you mind taking a look at it and updating it?